book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,472 @@
+Consider asking your speaker to break their talk into sections to allow for questions at different points in the session, or if your event is a lecture, consider using software to gather questions as the session progresses. 
+This can be helpful for people with lived experience of cognitive or sensory differences. 
+
+(cl-inclusive-events-comms-attendees)=
+## Who will be attending
+People with lived experience may experience additional anxiety due to being in an unfamiliar environment where they can often feel like their presence is tokenistic. 
+If you have planned your event well, attendees with lived experience should feel welcomed and able to engage fully in the event.
+
+It can be helpful in your communications to state who will be in attendance.
+For example "researchers, clinicians, policymakers, funding bodies, and people with lived experience". 
+
+Finally, consider including short introductions and background information about the speakers and session facilitators in your event information. 
+Many people with lived experience may feel more comfortable engaging in an event where they know who they are speaking with and it can be a helpful way to build rapport. 
+
+(cl-inclusive-events-comms-presentation-material)=
+## Presentation Slides and Event Content
+Although it is common in research to prepare presentation slides at the penultimate hour, to be as inclusive as possible, please work with your speakers and presenters to finalize session content before the event. 
+
+By working with speakers and requesting slides and presentation content ahead of the event, you will be best set up to support the involvement of people with disabilities and lived experience at your event. 
+
+This can help in two core ways, 1) supporting speaking in developing both psychologically and physically accessible content, and 2) ensuring that your event's content is accessible to a diverse audience.
+
+Firstly, consider how you will support speakers in developing accessible and inclusive material. 
+This may mean developing and sharing guidance on session content, or could also be giving feedback on presentation drafts. 
+Ensure that you contact and confirm your speakers with enough notice that allows them to prepare and revise material. 
+Please refer to the {ref}`cl-inclusive-events-comms-accessibility-guide` section for more recommendations on what to include in a speaker guidance pack for how to make presentations accessible. 
+
+Secondly, it is best practice to send out content before the session or event to ensure that people with sensory impairments, or who may require more time to review and engage with the material, are able to understand and take part. 
+
+If this is not possible, please address the reasons why in the opening address of the event and make alternative arrangements.
+For example, if the information is sensitive and unable to be shared due to Intellectual Property or Privacy reasons. 
```

Comment:
Or due to speakers not being able to meet deadlines, pfahhh. Sorry, not helpful comment :)

Reply:
Totally agree. 

Previous drafts were a bit more "because **some people** don't hand in their homework on time" 👨‍🎓 👀 but I thought that it was a bit too heavy-handed and was having a day where I believed the best in the world 😅